vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/java/com/vaadin/flow/component/shared/ClientValidationUtil.java

```diff
@@ -23,13 +23,21 @@
  * to valid as a result of client-side validation.
  *
  * @author Vaadin Ltd
+ * @deprecated since 24.8, use
+ *             {@code getElement().setProperty("manualValidation", true)}
+ *             instead.
```

Comment:
Sorry for commenting on a closed PR, but shouldn't here and in other places the classes have the `@Deprecated` annotation ?

Reply:
Good catch, thanks! Here is a fix: https://github.com/vaadin/flow-components/pull/7251